_includes/partials/connect.html

```diff
@@ -6,11 +6,11 @@
         
         <p class="community-block">
             <ul class="project-block">
-                <li><em>Join</em> <a href="https://docs.google.com/document/d/1c07UO9dS7_tFD-ClCWHIrEzRnzUJoFQ10EzfJTpS7FY/edit?usp=sharing">weekly community meeting</a> on <a href="/assets/projects/meshery/Meshery-Community-Meeting.ics">Fridays from 10am to 11am Central</a>.</li>
+                <li><em>Join</em> <a href="https://docs.google.com/document/d/1GrVdGHZAYeu6wHNLLoiaKNqBtk7enXE9XeDRCvdA4bY/edit?usp=sharing">meshery-ci weekly community meeting</a> on <a href="/assets/projects/meshery/Meshery-Community-Meeting.ics">Mondays from 10am to 11am Central</a>.</li>
```

Comment:
@Saim-Safdar I would suggest adding a new meeting, rather than renaming community meeting to CI meeting.

Reply:
@kushthedude Yes great suggestion I'll do that.